src/Fieldtypes/Terms.php

```diff
@@ -373,7 +373,7 @@ public function usingSingleTaxonomy()
     protected function createTermFromString($string, $taxonomy)
     {
         $term = Facades\Term::make()
-            ->slug(Str::slug($string))
+            ->slug(Str::slug($string, '-', Site::selected()->handle()))
```

Comment:
You probably want `lang()` instead of `handle()`.

Reply:
Yeah of course - fixed